src/mono/mono/mini/monovm.c

```diff
@@ -226,6 +228,90 @@ monovm_initialize (int propertyCount, const char **propertyKeys, const char **pr
 	return 0;
 }
 
+void mono_extract_and_register_properties (const char *buffer, long fileSize);
+
+// Initialize monovm with properties set by runtimeconfig.json. Primarily used by mobile targets.
+int
+monovm_runtimeconfig_initialize (MonovmRuntimeConfigArguments *arg, MonovmRuntimeConfigArgumentsCleanup cleanup_fn, void *user_data)
+{
+	switch (arg->kind) {
+	case 0: {
+		FILE *filePtr = NULL;
+		long fileSize = 0;
+		char *buffer = NULL;
+		size_t result;
+
+		filePtr = fopen(arg->runtimeconfig.name.path, "rb");
```

Comment:
I think we should use `mmap` when it is available instead of always copying the data.
See `mono/utils/mono-mmap.h` and `mono_file_map_open` and `mono_file_map`.

Reply:
Fixed. Thanks!